pkg/motor/interface.go

```diff
@@ -37,6 +37,7 @@ type MotorNode interface {
 	CreateBucket(context.Context, mt.CreateBucketRequest) (bucket.Bucket, error)
 	QueryWhereIs(ctx context.Context, did string) (mt.QueryWhereIsResponse, error)
 	QueryWhereIsByCreator(ctx context.Context) (mt.QueryWhereIsByCreatorResponse, error)
+	QueryBucketBySchema(context.Context, mt.QueryBucketContentBySchemaRequest) (mt.QueryBucketContentBySchemaResponse, error)
```

Comment:
This can be consolidated into the generic query request found in the motor protobuf file, after updating the branch